///@brief Enable Mainwindow transparency. Needs Windows >=2k; Introduce writeable Property window.transparency
diff -Nurw scintilla-scite/win32/SciTEWin.cxx scite/win32/SciTEWin.cxx
--- scintilla-scite/win32/SciTEWin.cxx	2016-12-17 18:53:36.000000000 +0100
+++ scite/win32/SciTEWin.cxx	2016-12-03 13:55:12.000000000 +0100
@@ -1356,12 +1356,14 @@
 	        (left != static_cast<int>(CW_USEDEFAULT))) {
 		left += width;
 	}
+
 	// Pass 'this' pointer in lpParam of CreateWindow().
+	//  Windows >=2k allows transparency for any main hwnd. >=win8 for childs too usin WS_EX_LAYERED / OVERLAPPEDWINDOW
 	wSciTE = ::CreateWindowEx(
-	             0,
+	             WS_EX_LAYERED,
 	             className,
 	             windowName.c_str(),
-	             WS_CAPTION | WS_SYSMENU | WS_THICKFRAME |
+	             WS_OVERLAPPEDWINDOW | WS_CAPTION | WS_SYSMENU | WS_THICKFRAME |
 	             WS_MINIMIZEBOX | WS_MAXIMIZEBOX |
 	             WS_CLIPCHILDREN,
 	             left, top, width, height,
@@ -1374,6 +1374,12 @@
 
 	if (props.GetInt("save.position"))
 		RestorePosition();
+	// ensure a minimum initial value.
+	if (props.GetInt("window.transparency")>89) {
+		SetLayeredWindowAttributes(HwndOf(wSciTE), 0, (255 *  (props.GetInt("window.transparency"))) / 100, LWA_ALPHA);
+	} else {
+		SetLayeredWindowAttributes(HwndOf(wSciTE), 0, (255 * 100) / 100, LWA_ALPHA);
+	}
 
 	LocaliseMenus();
 	std::string pageSetup = props.GetString("print.margins");
